deltachat-jsonrpc/src/api/types/message.rs

```diff
@@ -104,6 +104,9 @@ enum MessageQuote {
     WithMessage {
         text: String,
         message_id: u32,
+        /// The quoted message does not always belong
+        /// to the parent message's chat, e.g. when "Reply Privately" is used.
```

Comment:
```suggestion
        /// The quoted message does not always belong
        /// to the same chat, e.g. when "Reply Privately" is used.
```

Reply:
OK yeah, I meant "parent message" as "the message that contains the quote".